src/DotNetWorker.Grpc/protobuf/src/proto/FunctionRpc.proto

```diff
@@ -282,12 +296,15 @@ message FunctionsMetadataRequest {
 }
 
 // Worker sends function metadata back to host
-message FunctionMetadataResponses {
+message FunctionMetadataResponse {
   // list of function indexing responses
-  repeated FunctionLoadRequest function_load_requests_results = 1;
+  repeated RpcFunctionMetadata function_metadata_results = 1;
 
   // status of overall metadata request
   StatusResult result = 2;
+
+  // if set to true then host will perform indexing
+  bool useDefaultMetadataIndexing = 3;
```

Comment:
nit: I see other props use `_` (ex: `function_metadata_results`) and this one use camelCase. Should we use the same format to be consistent?

Reply:
Good catch - this is a Stein change so fixing it won't break anything existing -- the change will have to come from the [protobuf repo](https://github.com/Azure/azure-functions-language-worker-protobuf) though, will talk to @surgupta-msft offline about this.